src/autoscaler/metricscollector/cmd/metricscollector/metricscollector_test.go

```diff
@@ -284,4 +285,21 @@ var _ = Describe("MetricsCollector", func() {
 			rsp.Body.Close()
 		})
 	})
+
+	Context("when using streaming for metrics collection", func() {
```

Comment:
Maybe this should be a Describe and we need another above sections "When using polling for metrics collection". And then, a context "when a request for metrics history comes"on top of "It("returns with a 200 for metric query", func() " done below.